CONTRIBUTING.md

```diff
@@ -106,3 +126,7 @@ Build requires [Ruby](https://www.ruby-lang.org/en/). Under the hood [Browserify
 To build simply run
 
     $ ./build
+
+
+[ES5]: http://www.ecma-international.org/ecma-262/5.1/
+[ES6][ES6]: http://www.ecma-international.org/ecma-262/6.0/
```

Comment:
My markdown foo may not be the best (https://github.com/sinonjs/sinon/commit/3b15a28219e884fbdcce5c1b24563c4250926451), but is the double `[ES6]` intended?